src/search/csearch.c

```
@@ -62,117 +111,100 @@ int main(int argc, char *argv[])
-        if (i_arg < argc)
-          delta = (real)atof(argv[i_arg]);
-        else 
-        {
-          #ifdef ERROR
-          fprintf(STDERR,"*** error (SEARCH): initial displacement value not given\n");
-          #endif
-          exit(1);
-        }
-      }
+    /* Read initial displacement */
+    if (strncmp(argv[i_arg], "-d", 2) == 0) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)